trade-finance-manager-ui/component-tests/assertions.js

```diff
@@ -1,5 +1,7 @@
 // eslint-disable-next-line no-unused-vars
 const assertions = (wrapper, html, params) => ({
+  html,
+  params,
```

Comment:
Allows us to access these in tests. I needed the params for one of my tests, but having `html` also makes debugging easier.